content/pt-br/docs/concepts/policy/pid-limiting.md

```diff
@@ -0,0 +1,123 @@
+Além disso, é importante garantir que os PIDs sejam limitados entre os Pods para
+para garantir que tenham impacto limitado em outras cargas de trabalho no mesmo nó.
+
+{{< note >}}
+Em certas instalações do Linux, o sistema operacional define o limite de PIDs para um padrão 
+como por exemplo `32768`. Considere aumentar o valor de  `/proc/sys/kernel/pid_max`.
+{{< /note >}}
+
+Você pode configurar um kubelet para limitar o número de PIDs que um determinado pod pode consumir.
+Como por exemplo se o sistema operacional host do seu nó estiver configurado para usar um máximo de `262144` PIDs e
+estiver experado hospedar menos de `250` Pods, pode-se dar a cada Pod um orçamento de `1000`
+PIDs para evitar o uso do número geral de PIDs disponíveis desse nó. Se o
+administrador deseja sobrecarregar PIDs semelhantes à CPU ou memória, eles também podem fazer isso
+com alguns riscos adicionais. De qualquer forma, um único Pod não será capaz de trazer
+toda a máquina para baixo. Este tipo de limitação de recursos ajuda a evitar simples
+fork bombs de afetar uma operação de um cluster inteiro.
+
+A limitação de PID por pod permite que os administradores protejam um pod de outro, mas
+não garante que todos os pods programados nesse host não possam afetar o nó em geral.
+A limitação por pod também não protege os próprios agentes do nó do esgotamento do PID.
+
+Você também pode reservar uma quantidade de PIDs para sobrecarga do nó, separada da
+alocação para os pods. Isso é semelhante a como você pode reservar uma CPU, memória ou outros
+recursos para uso pelo sistema operacional e outras instalações fora dos Pods
+e seus recipientes.
+
+A limitação de PID  é um importante irmão para [compute
+resource](/pt-br/docs/concepts/configuration/manage-resources-containers/) requests
+e limites. No entanto, você o especifica de uma maneira diferente: em vez de definir um
+limite de recursos do pod no `.spec` para um pod, você configura o limite como uma
+configuração no kubelet. Os limites de PID definidos pelo pod não são suportados no momento.
+
+{{< caution >}}
+Isso significa que o limite aplicável a um pod pode ser diferente dependendo
+onde o pod está alocado. Para tornar as coisas simples, é mais fácil se todos os nós usarem
+os mesmos limites e reservas de recursos PID.
+{{< /caution >}}
+
+## Limites de PID de nós
+
+O Kubernetes permite que você reserve vários IDs de processo para uso do sistema. Para
+configurar a reserva, use o parâmetro `pid=<número>` no
+opções de linha de comando `--system-reried` e `--kube-reried` para o kubelet.
+O valor especificado declara que o número especificado de IDs de processo será
+reservado para o sistema como um todo e para os daemons do sistema Kubernetes
+respectivamente.
+
+{{< note >}}
+Antes da versão 1.20 do Kubernetes, limitação de recursos PID com nível de nós
+reservas necessárias habilitando o [feature-gates
+](/docs/reference/command-line-tools-reference/feature-gates/)
+`SupportNodePidsLimit` para funcionar.
```

Comment:
This paragraph needs some review. The original meaning is not really being conveyed here (that a user would need to turn the feature gate on to have access to PID limiting).

It would be good to stylize feature gate as italic too.

Reply:
Thanks for the insight, i changed but now in the original document they deleted the notes, so now in this doc i have to remove too?

Reply:
Hi @DaviMatheus! Yes, that is correct.